src/app/cli/src/commands/system_api_server_run_command.rs

```diff
@@ -63,6 +75,28 @@ impl Command for APIServerRunCommand {
     async fn run(&mut self) -> Result<(), CLIError> {
         // Check required env variables are present before starting API server
         self.check_required_env_vars()?;
+        let current_account_name = match self.account_subject.as_ref() {
+            CurrentAccountSubject::Logged(l) => l.account_name.clone(),
+            CurrentAccountSubject::Anonymous(_) => {
+                unreachable!("Cannot launch API server with anonymous account")
+            }
+        };
+        let login_credentials = accounts::PasswordLoginCredentials {
+            login: current_account_name.to_string(),
+            // Note: note a mistake, use identical login and password, equal to account name
+            password: current_account_name.to_string(),
+        };
+
+        let auth_svc = self.catalog.get_one::<dyn AuthenticationService>().unwrap();
+        let access_token = auth_svc
+            .login(
+                &accounts::LOGIN_METHOD_PASSWORD.to_string(),
+                serde_json::to_string::<accounts::PasswordLoginCredentials>(&login_credentials)
+                    .int_err()?,
+            )
+            .await
+            .int_err()?
+            .access_token;
```

Comment:
The method `run` is getting a bit long, it basically doubled. Perhaps sub-methods for the large parts would make it more readable. At least a private method for the the new code.